src/app/components/projects/projects.menus.ts

```diff
@@ -111,8 +112,8 @@ export const uploadAnnotationsProjectMenuItem = menuRoute({
   icon: ["fas", "file-import"],
   label: "Batch Upload Annotations",
   parent: projectMenuItem,
-  // TODO Change to isProjectEditorPredicate
-  predicate: isAdminPredicate,
+  // TODO: Once functionality is implemented, this should be changed to isProjectEditorPredicate
```

Comment:
should it?

you should rely on a capability from the server if it exists...

Reply:
From my understanding, this is how the `isProjectEditorPredicate` predicate works

![image](https://github.com/QutEcoacoustics/workbench-client/assets/33742269/dc5993e3-e10b-4801-bf7d-4999763cd9e5)

Where `canEdit` comes from the server capabilities

![image](https://github.com/QutEcoacoustics/workbench-client/assets/33742269/6569076a-8452-443b-aaee-ec34ee9e7bab)

Note: Because this PR is going to be merged before the annotation uploads, I've put this behind the `wipPredicate`

Reply:
My point rather is that assuming someone can edit a project means that they can upload annotations is not correct.

From my memory of the servers permissions: only owners can edit projects but anyone with writer can upload annotations.

Reply:
> My point rather is that assuming someone can edit a project means that they can upload annotations is not correct.
> 
> From my memory of the servers permissions: only owners can edit projects but anyone with writer can upload annotations.

Understandable, I've backfilled a new predicate in preparation

![image](https://github.com/QutEcoacoustics/workbench-client/assets/33742269/47dadc96-45aa-4925-9772-a1645c0b6b80)